jaxws-retail/service/pom.xml

```diff
@@ -39,6 +39,7 @@
     </licenses>
     <properties>
         <checkstyle.skip>true</checkstyle.skip>
+        <version.jbossws-cxf-client>6.1.0.Final</version.jbossws-cxf-client>
```

Comment:
Perhaps we should we also add this to the JAXWS Client BOM?

Reply:
I have no objection

Reply:
it is already there btw https://repo1.maven.org/maven2/org/wildfly/wildfly-jaxws-client-bom/27.0.1.Final/wildfly-jaxws-client-bom-27.0.1.Final.pom

Reply:
Indeed, just confirmed such artifact is a top level dependency on the JAXWS-Client BOM (see https://github.com/wildfly/boms/blob/main/client/jaxws-client/pom.xml#L100 ), so no need to manage version here

Reply:
org.jboss.ws.plugins:jaxws-tools-maven-plugin does not process wildfly-jaxws-client-bom
as the enclosing pom.xml does, thus org.jboss.ws.cxf:jbossws-cxf-client must be specified
with version in order for wsconsume to function properly.  Declaring the BOM and/or
jbossws-cxf-client in a dependency block within jaxws-tools-maven-plugin does not work
either.  It appears the declaration of jbossws-cxf-client must stay.